server/src/test-utils/mocks/application/mock-create-user.ts

```diff
@@ -0,0 +1,15 @@
+import { MockUserRepo } from '../../../modules/users/infra/repos/implementations/mock-user-repo'
+import { CreateUserUseCase } from '../../../modules/users/application/use-cases/create-user/create-user-use-case'
+import { CreateUserController } from '../../../modules/users/application/use-cases/create-user/create-user-controller'
+import { User } from '../../../modules/users/domain/entities/user'
+import { UserMap } from '../../../modules/users/mappers/user-map'
+
+const mockCreateUser = async (users: Array<User> = []) => {
```

Comment:
```suggestion
const buildCreateUserMocks = async (users: Array<User> = []) => {
```

or something along those lines. `buildMocksForCreateUser` is another option?